src/devicecontroller/DefaultDeviceController.ts

```diff
@@ -676,6 +676,10 @@ export default class DefaultDeviceController implements DeviceControllerBasedMed
     kind: string,
     device: Device
   ): MediaStreamConstraints | null {
+    let supportedConstraints = navigator.mediaDevices.getSupportedConstraints();
+    this.logger.warn(
```

Comment:
- Log only once in the constructor since the list won't change
- Use `info()` as it's not a warning or error
- nit: Use `const`
- nit: Use template strings (``) over the plus sign (+) for concatenation